.profile.d/heroku-metrics-daemon.sh

```diff
@@ -37,7 +37,7 @@ if [[ -f build.sbt ]]; then
     unzip -qq bin/heroku-metrics-agent.jar 'javax/*' -d bin/ext/
     export JAVA_OPTS="-javaagent:bin/heroku-metrics-agent.jar=cp=/app/bin/ext/ -Xbootclasspath/a:bin/heroku-metrics-agent.jar ${JAVA_OPTS}"
     AGENTMON_FLAGS+=("-prom-url=http://localhost:${HEROKU_METRICS_PROM_PORT}${HEROKU_METRICS_PROM_ENDPOINT}")
-elif [[ -f pom.xml ]] || [[ -f build.gradle ]] || [[ -d .jdk ]]; then
+elif [[ -f pom.xml ]] || [[ -f build.gradle ]] || [[ -f project.clj ]] || [[ -d .jdk ]]; then
```

Comment:
There is no "return code" as such in Bash. only exit codes. So you can't return a string, which is why you see functions echo a string and then the caller captures it with s="$(my_func)". bash is cra cra. I'll submit another PR with some refactoring changes like this.